lib/src/app/player.js

```diff
@@ -93,6 +94,13 @@ class CastPlayer {
     if (typeof data.params === 'object') {
       Object.assign(params, data.params);
     }
+
+    // we need to remove the 'initialTime' param from the previous asset
+    // only if it was not set or is equal to zero.
+    if ( get(data.params, 'initialTime', 0) === 0) {
```

Comment:
Since the main purpose of `lodash.get` (deep property lookup) is not used here, let me suggest here to remove 4th line and use just:
`if (!data.params.initialTime) {`
Less deps, less functions signatures keeping in mind, less price of code maintenance.